### PR TITLE
urdfdom 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by jslee02**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

The URDF (U-Robot Description Format) library provides core data structures and a simple XML parsers for populating the class data structures from an URDF file.

Repo: https://github.com/ros/urdfdom

Depends on #23901 and #23899

### CODE REVIEW

**Comment by woodruffw** [comment]
```
Warning: Found Ruby file outside homebrew/core tap formula directory
(/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula):
  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/urdfdom.rb
```

File: Formula/urdfdom.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          "  <link name='link_0'>  "
+          "  </link>               "
+          "</robot>                ";
+        urdf::parseURDF(xml_string);
+        return 0;
+      }
+    EOS
+    system ENV.cc, "test.cpp", "-L#{lib}", "-lurdfdom_world", "-lc++", "-std=c++11", "-o", "test"
+    system "./test"
+  end
```
Comment (fxcoudert) [inline Formula/urdfdom.rb]: Great test!

File: Formula/urdfdom.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+class Urdfdom < Formula
+  desc "URDF parser"
+  homepage "https://wiki.ros.org/urdf"
+  url "https://github.com/ros/urdfdom/archive/1.0.0.tar.gz"
+  sha256 "243ea925d434ebde0f9dee35ee5615ecc2c16151834713a01f85b97ac25991e1"
+
+  depends_on "cmake" => :build
+  depends_on "tinyxml"
+  depends_on "console_bridge"
+  depends_on "urdfdom_headers"
```
Comment (fxcoudert) [inline Formula/urdfdom.rb]: Alphabetical ordering of the runtime deps please

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Great!

**Pull request closed without merging**